source/geometry/FloatConvexHull2D.ooc

```diff
@@ -25,9 +25,7 @@ FloatConvexHull2D: class {
 			this computeHull()
 	}
 	init: func ~fromArray (array: FloatPoint2D[], computeHull := true) {
-		list := VectorList<FloatPoint2D> new(array length)
-		for (i in 0 .. array length)
-			list add(array[i])
+		list := VectorList<FloatPoint2D> new(array)
```

Comment:
Are you sure the correct constructor is called, like ooc is not interpreting "array" as Pointer, converting to Int and using this
```ooc
init: func ~heap (capacity: Int, freeContent := true) {
 		this init(HeapVector<T> new(1 maximum(capacity), freeContent))
 	}
```
instead ?